]2;feyrer@yui:/usr/pkgsrc/sysutils/top/work/top-3.5beta7 ]1;yui--- Makefile.X.orig	Wed Nov 20 03:19:24 1996
+++ Makefile.X	Fri Feb 20 08:11:35 1998
@@ -55,6 +55,12 @@
 TERMCAP = %termcap%
 
 CFLAGS = %cflgs% $(CDEFS)
+.ifndef UVM
+UVM=	
+.endif
+.if ${UVM} != ""
+CFLAGS+=        -DUVM
+.endif
 LINTFLAGS = -x $(CDEFS)
 
 all: Makefile top.local.h top
@@ -100,10 +106,10 @@
 install: top top.1 install-top install-$(MANSTY)
 
 install-top:
-	$(INSTALL) -o $(OWNER) -m $(MODE) -g $(GROUP) top $(BINDIR)
+	${INSTALL} ${COPY} ${STRIPFLAG} -o $(OWNER) -m $(MODE) -g $(GROUP) top $(BINDIR)
 
 install-man:
-	$(INSTALL) top.1 $(MANDIR)/top.$(MANEXT)
+	${INSTALL_MAN} top.1 $(MANDIR)/top.$(MANEXT)
 
 install-catman:
 	tbl top.1 | nroff -man > $(MANDIR)/top.$(MANEXT)
